tests/client/apiClient/loginUser.test.js

```diff
@@ -0,0 +1,21 @@
+import nock from 'nock'
+
+import {loginUser} from '../../../client/apiClient'
+
+const userDetails = {
+  id: 1,
+  name: 'Janie',
+  email: 'janie@test.com',
```

Comment:
I don't think we ever ask for email address. :wink: Even though we're mocking out the API response, let's make sure the test data matches what we get from the live API. The object returned from the API looks like this:
```
{
  "message": "Authentication successful.",
  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwidXNlcm5hbWUiOiJkb24iLCJpYXQiOjE1MjU2NjQ4NTgsImV4cCI6MTUyNTc1MTI1OH0.ofOMvjwHAEkbAnMK7NC2xG3RneUlGGeTE52OC-Di06w"
}
```